src/Libraries/Builtin/Builtin.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We also have a `BuiltIn.ds` file that wraps calls to `List` builtin methods inside a `DS` `List` class. So this could be a little confusing having another project named `Builtin`. Minor point since we may get rid of the `BuiltIn.ds` eventually, I guess.

Reply:
I think this is fine. I think it's clear that one contains builtin DesignScript code and the other is builtin code contained in an assembly.